docs/add_content_and_edit_views.md

```diff
@@ -0,0 +1,268 @@
+            '.ez-tweet-input-ui input': {
+                'blur': 'validate',
+                'valuechange': 'validate'
+            }
+        },
+
+        /**
+         * Validates the current input of tweet
+         *
+         * @method validate
+         */
+        validate: function () {
+            var validity = this._getInputValidity();
+
+            if ( validity.typeMismatch || validity.patternMismatch ) {
+                this.set('errorStatus', Y.eZ.trans('url.not.valid', {}, 'fieldedit'));
+            } else if ( validity.valueMissing ) {
+                this.set('errorStatus', Y.eZ.trans('this.field.is.required', {}, 'fieldedit'));
+            } else {
+                this.set('errorStatus', false);
+            }
+        },
+
+        /**
+         * Defines the variables to be imported in the field edit template for tweet.
+         *
+         * @protected
+         * @method _variables
+         * @return {Object} containing isRequired
+         * entries
+         */
+        _variables: function () {
+            var def = this.get('fieldDefinition');
+
+            return {
+                "isRequired": def.isRequired
+            };
+        },
+
+        /**
+         * Returns the input validity state object for the input generated by
+         * the tweet template
+         *
+         * See https://developer.mozilla.org/en-US/docs/Web/API/ValidityState
+         *
+         * @protected
+         * @method _getInputValidity
+         * @return {ValidityState}
+         */
+        _getInputValidity: function () {
+            return this.get('container').one('.ez-tweet-input-ui input').get('validity');
+        },
+
+        /**
+         * Returns the currently filled tweet value
+         *
+         * @protected
+         * @method _getFieldValue
+         * @return String
+         */
+        _getFieldValue: function () {
+            return this.get('container').one('.ez-tweet-input-ui input').get('value');
+        }
+    });
+
+    Y.eZ.FieldEditView.registerFieldEditView(
+        FIELDTYPE_IDENTIFIER, Y.eZ.TweetEditView
+    );
+});
+```
+
+### Registering the view
+
+We also have to create the YML file where we will tell the system to use our files. The example configuration looks like this:
+
+```yaml
+//Resources/config/yui.yml
+
+system:
+    default:
+        yui:
+            modules:
+                tweeteditview-ez-template:
+                    type: 'template'
+                    path: bundles/ezsystemstweetfieldtype/templates/fields/edit/tweet.hbt
+                ez-tweet-editview:
+                    requires: ['ez-fieldeditview', 'event-valuechange', 'tweeteditview-ez-template']
+                    dependencyOf: ['ez-contenteditformview']
+                    path: bundles/ezsystemstweetfieldtype/js/views/fields/ez-tweet-editview.js
+```
+
+We need to prepend this configuration, like we did with our template. To do this, we have to edit dependency injection extension:
```

Comment:
Adding some link to this topic from our Documentation?

Reply:
It isn't documented in our documentation since it's a Symfony feature. Also, it was already done in the previous step, here we are just adding next config file.
I will update the code snippet to include the whole class so it's more clear what is edited.